magento/magento2
# Incorrect http method issue fix

### Description (*)
Incorrect http method class removed



### Fixed Issues (if relevant)

1. Fixes #39261 

### Manual testing scenarios (*)


1. Go to REPORTS > Products > Ordered
2. Select a date range for which the product was sold.
3. Click to Export csv or excel
4. Verify if the file is exported or not.



### Contribution checklist (*)
